File: Golang/1009.go
package main

import (
	"fmt"
)

func main() {

	var nome string
	var salario_fixo, total_vendas float64

	fmt.Scan(&nome)
	fmt.Scan(&salario_fixo)
	fmt.Scan(&total_vendas)

	salario := salario_fixo + (total_vendas * 0.15)

	fmt.Printf("TOTAL = R$ %.2f\n", salario)
}

File: Elixir/1009.exs
id_funcionario = IO.gets("") |> String.trim()
salario_fixo = IO.gets("") |> String.trim() |> String.to_float
vendas = IO.gets("") |> String.trim() |> String.to_float

salario = (salario_fixo + (vendas * 0.15))

IO.puts "TOTAL = R$ #{:erlang.float_to_binary(salario, decimals: 2)}"
